demo/build.gradle

```diff
@@ -16,9 +16,20 @@ import org.apache.tools.ant.filters.ConcatFilter
  * limitations under the License.
  */
 
-apply plugin: 'com.android.application'
+plugins {
+    id 'com.android.application'
+    id 'com.google.secrets_gradle_plugin' version '0.4'
+}
 
 android {
+    signingConfigs {
+        tesing {
```

Comment:
Looks like this was left over from testing?

Reply:
Ah, yes. Will remove.